src/domain/communication/discussion/pages/DiscussionPage.tsx

```diff
@@ -20,28 +20,31 @@ import { useUrlParams } from '../../../../core/routing/useUrlParams';
 import TopLevelPageLayout from '../../../../main/ui/layout/topLevelPageLayout/TopLevelPageLayout';
 import RouterLink from '../../../../core/ui/link/RouterLink';
 import BackButton from '../../../../core/ui/actions/BackButton';
-import { useLocation, useNavigate } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
 import usePostMessageMutations from '../../room/Comments/usePostMessageMutations';
 import useSubscribeOnRoomEvents from '../../../collaboration/callout/useSubscribeOnRoomEvents';
 import { ForumOutlined } from '@mui/icons-material';
 import BreadcrumbsItem from '../../../../core/ui/navigation/BreadcrumbsItem';
 import TopLevelPageBreadcrumbs from '../../../../main/topLevelPages/topLevelPageBreadcrumbs/TopLevelPageBreadcrumbs';
 import UpdateDiscussionDialog from '../views/UpdateDiscussionDialog';
 import { StorageConfigContextProvider } from '../../../storage/StorageBucket/StorageConfigContext';
+import useNavigate from '../../../../core/routing/useNavigate';
 
-interface DiscussionPageProps {}
+interface DiscussionPageProps {
+  releaseId?: string;
+}
 
-export const DiscussionPage: FC<DiscussionPageProps> = () => {
+export const DiscussionPage: FC<DiscussionPageProps> = ({ releaseId }) => {
```

Comment:
I would suggest moving discussionNameId to be a prop that is passed. 
We will then have:
In Route.tsx <Route path={`discussion/:${nameOfUrl.discussionNameId}`} element={<DiscussionPage discussionNameId={nameOfUrl.discussionNameId} />} />
And here we would not bother to check whether to use releaseId or discussionNameId.